source/docs/advanced/performance/preloading-routes.md

```diff
@@ -0,0 +1,183 @@
+---
+id: preloading-routes
+title: Preloading Routes' data
+---
+
```

Comment:
I think we should add this here for the whole page for now. At some point we could remove all those notices when we will be confident that all projects use 1.0 for instance (and later, after EOL).

```suggestion

<blockquote class="feature--new">
  _This feature has been added in version `1.0.0-rc.0`_
</blockquote>

```